DietPi-Software | O!MPD

Joulinar opened the pull request:
There is an issue report on the forum about a missing `/` in `config.local.inc.php` configuration file

**Status**: Ready
- [x] adjust config file

**Reference**: https://dietpi.com/phpbb/viewtopic.php?t=8904

**Commit list/description**:
- DietPi-Software | O!MPD - adjust config file

Joulinar committed:
v7.1

DietPi-Software | O!MPD - adjust config file

MichaIng commented:
Many thanks!

Interesting, it seems in the past a slash was added: https://github.com/ArturSierzant/OMPD/commit/f1a09c23839ef4b9e18e1ad72da8083982d94376#diff-704e201ceeca11d129bcd3cc6428dbfb66231f63151d02ba7c976f56a731b1ee

Strange that the library works pretty well without the slash but only the browser not. However, easy fix.

MichaIng committed:
v7.1

+ CHANGELOG | O!MPD: Resolved an issue where browsing the media directory from the web UI failed
